roles/servicetelemetry/tasks/component_prometheus.yml

```diff
@@ -87,7 +132,7 @@
       roleRef:
         apiGroup: rbac.authorization.k8s.io
         kind: ClusterRole
-        name: prometheus-k8s
+        name: prometheus-stf
```

Comment:
Same issue as above where roleRef's cannot be changed after the fact.

This isn't found in upstream CI on CRC because the clusters are (presumably) created fresh every time, but our existing Jenkins-based deployments finds it since I guess we're not (properly?) cleaning out all the ClusterRoleBindings before doing a fresh deployment?

Reply:
Fixed via #467